forge.lic

```diff
@@ -75,7 +75,7 @@ class Forge
 
   def temper
     unless checkleft =~ /#{@item}/i || checkright =~ /#{@item}/i
-      echo('please hold the item to temper')
+      echo('***Please hold the item to temper.***')
```

Comment:
Honestly I'm fine with not needing a noun for temper/hone/balance. Just do the action on whatever you're holding. That's how sewing works for reinforcing or sealing.